geomesa-convert/geomesa-convert-text/src/main/scala/org/locationtech/geomesa/convert/text/DelimitedTextConverter.scala

```diff
@@ -79,6 +100,7 @@ class DelimitedTextConverter(format: CSVFormat,
   }
 
   override def close(): Unit = {
+    es.shutdown()
```

Comment:
I think shutdown won't terminate the running thread, so it will just keep looping/polling. You should use shutdownNow() or add some termination logic to the writing thread.